src/core/util/log.cc

```diff
@@ -64,12 +65,62 @@ namespace kovri
 {
 namespace core
 {
+namespace logging = boost::log;
+namespace expr = boost::log::expressions;
+namespace attrs = boost::log::attributes;
+
+
+std::string SetColor(
+    logging::value_ref<logging::trivial::severity_level,
+    logging::trivial::tag::severity> const& severity, bool color)
+{
+  if (!color)
+  {
+    return "";
+  }
+  else if (severity)
+  {
+    switch (severity.get())
+    {
+      case logging::trivial::fatal:
+      case logging::trivial::error:
+        return "\033[31m"; // Red
+      case logging::trivial::info:
+        return "\033[32m"; // Green
+      case logging::trivial::warning:
+        return "\033[33m"; // Yellow
+      case logging::trivial::debug:
+        return "\033[34m"; // Blue
+      case logging::trivial::trace:
+        return "\033[36m"; // Cyan
+    }
+  }
+  return "\033[0m"; // Reset
+}
+
+logging::formatter GetFormat(bool color)
+{
+  logging::formatter fmt = 
+      expr::stream << boost::phoenix::bind(
+                          &SetColor, logging::trivial::severity.or_none(), color)
+                   << "["
+                   << expr::format_date_time(
+                          expr::attr<boost::posix_time::ptime>("TimeStamp"),
+                          "%Y.%m.%d %T.%f")
+                   << "]"
+                   << " [" << expr::attr<attrs::current_thread_id::value_type>(
+                                  "ThreadID")
+                   << "]"
+                   << " [" << logging::trivial::severity << "]"
+                   << "  " << expr::smessage
+                   << (color == true ? "\033[0m" : ""); // Reset
+
+  return fmt;
```

Comment:
As noted in our [style guide](https://github.com/monero-project/kovri-docs/blob/master/i18n/en/style.md), will you please consider the following?:

- running `clang-format` on all of your two new functions
- as noted in [Naming](https://google.github.io/styleguide/cppguide.html#Naming), `color` and `fmt` could be more descriptive (e.g., `has_color` and `format`)

Also, including this PR number in the commit log (e.g., `Referencing #729` or similar).

Reply:
Improved variable names and ran `clang-format` on my functions.

>Also, including this PR number in the commit log (e.g., Referencing #729 or similar).

Asked you about that on IRC.